src/Administration/Resources/app/administration/src/module/sw-settings-media/snippet/en-GB.json

```diff
@@ -4,6 +4,12 @@
       "title": "Media",
       "description": "Manage media settings here.",
       "buttonSave": "Save"
+    },
+    "3d": {
+      "lightIntensity": {
+        "label": "Light Intensity",
+        "helpText": "Change the intensity of the light in the 3D view."
```

Comment:
I would change it to "Adjust the light intensity in 3D view.", of course only a suggestion, approving the pr in any case 👍